src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private final double high; //Add random parameters of Figure
+    private final double firstSide;
+    private final double secondSide;
+
+    private final Color color;
```

Comment:
Consider reordering the fields of the IsoscelesTrapezoid class to group related fields together for better readability. Typically, the color field would be placed after all the geometric properties (high, firstSide, secondSide).